R/utils.R

```diff
@@ -100,3 +100,23 @@ add_profile_ranking_average <- function(data, product) {
 
   data |> left_join(profile_ranking_average, by = c("companies_id", "grouped_by"))
 }
+
+check_matches_col_names <- function(data, cols) {
+  if (!any(cols %in% names(data))) {
+    pattern <- cols[!(cols %in% names(data))]
+    abort(c(
+      glue("The data lacks column '{pattern}'."),
+      i = "Are you using the correct data?"
+    ), class = "check_matches_name")
+  }
+  invisible(data)
+}
```

Comment:
You may instead reuse `check_matches_name()`

https://github.com/2DegreesInvesting/tiltIndicatorAfter/blob/main/R/import-standalone-name-pattern.R

Reply:
To use `check_matches_name()` we need to export this function in tiltIndicator. Should I do that with a separate ticket?

Reply:
The links above show we already use this function in tilt indicatorAfter. And the link immediately above points you to the exact file where it lives.

Reply:
My bad! Yeah right! I will adapt the code accordingly